static-parts/README-start.md

```diff
@@ -42,4 +42,10 @@ Do you have an excellent idea or know some cool questions that aren't on the lis
 
 Join our [Gitter channel](https://gitter.im/30-seconds-of-interviews/Lobby) to help with the development of the project.
 
+#### Related projects
+
+* [30 Seconds of CSS](https://30-seconds.github.io/30-seconds-of-css/)
+* [30 Seconds of Interviews](https://30secondsofinterviews.org/)
```

Comment:
er... I think this is supposed to be 30-seconds-of-code, not interviews haha

Reply:
Oh... I am soooo tired. Will fix this right now 👍